releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,15 @@
 
 [Image][image] | [Sample][sample]
 
+## 2.0.0 -
+
+### Enhancements
+* Added support for namespaced service account. It can establish federated identity credential between the workload identity and the namespaced service account subject, which is useful to control access to configuration between apps and namespaces.
+* Added support for collecting information about your use of Azure App Configuration Kubernetes Provider. Telemetry is enabled by default and can be disabled by setting `requestTracing.enabled` to `false` during installation.

```

Comment:
This is privacy relevant change, not sure this is the right way to annouce this change to users, I don't think it's an enhancement. We need to upate the privacy section in the README of k8s provider, and just tell user it can be disabled here.

Reply:
@zhenlan  thoughts？

Reply:
So in releaseNotes we don't cover telemetry part, right?